doc/classes/SceneTree.xml

```diff
@@ -82,26 +82,25 @@
 				}
 				[/csharp]
 				[/codeblocks]
-				The timer will be automatically freed after its time elapses.
 			</description>
 		</method>
 		<method name="create_tween">
 			<return type="Tween" />
 			<description>
-				Creates and returns a new [Tween].
+				Returns a newly created [Tween].
```

Comment:
Isn't the old description more accurate? Yours sounds like the Tween was created somewhere else.

Reply:
The intention was opposite. It was to hint that the Tween is tied to the SceneTree post-creation, so perhaps it is worth stating that specifically, instead.